modules/claims_api/lib/bd/bd.rb

```diff
@@ -123,16 +122,21 @@ def generate_upload_body(claim:, doc_type:, pdf_path:, file_number: nil, origina
     end
     # rubocop:enable Metrics/ParameterLists
 
-    def generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:)
+    def generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:, api_version:)
       # https://confluence.devops.va.gov/display/VAExternal/Document+Types
-      doc_type_to_form_name = {
-        'L075' => '21-22a',
-        'L122' => '526EZ',
-        'L190' => '21-22',
-        'L705' => '5103'
+      doc_type_names = {
+        'v1' => {
+          'L075' => 'representative'
```

Comment:
Is there ever a chance we could get `L190` through `v1`? E.g.,

https://github.com/department-of-veterans-affairs/vets-api/blob/b44baf2c9ca58e4124869fcedeb609e3f6804d21/modules/claims_api/app/sidekiq/claims_api/v1/poa_form_builder_job.rb#L23

Reply:
@tycol7 total oversight on my part, thanks for catching that!  [b2a6592](https://github.com/department-of-veterans-affairs/vets-api/pull/18259/commits/b2a6592dea28c24eadd9fe90f9af8b563da1cee7)